elixir_buildpack.config

```diff
@@ -1,4 +1,4 @@
 erlang_version=19.2
 elixir_version=1.4.0
 always_rebuild=true
-config_vars_to_export=(APPLICATION_ENVIRONMENT HOST_NAME ROLLBAR_ACCESS_TOKEN SECRET_KEY_BASE)
+config_vars_to_export=(HOST_NAME ROLLBAR_ACCESS_TOKEN SECRET_KEY_BASE)
```

Comment:
Nope. That config makes env vars available at compile time. Though `$DEPLOY_TAG` is evaluated at compile time, it is only used at runtime.